include/cutlass/gemm/kernel/intel_pvc_gemm.hpp

```diff
@@ -113,6 +113,16 @@ class GemmUniversal<
 
   static constexpr int VecC = CollectiveMainloop::VecC;
 
+  // Kernel level shared memory storage
+  struct SharedStorage {
+    // Mainloop and epilogue don't use smem concurrently since kernel is non-persistent, so we can use a union
+    union TensorStorage {
+      using EpilogueTensorStorage = typename CollectiveEpilogue::TensorStorage;
+
+      EpilogueTensorStorage epilogue;
+    } tensors;
```

Comment:
What are the union you are creating here? 
Did you wanted to put something like this?
```c++
struct SharedStorage {
    // Mainloop and epilogue don't use smem concurrently since kernel is non-persistent, so we can use a union
    union TensorStorage {
       using MainloopTensorStorage = typename CollectiveMainloop::TensorStorage;
       using EpilogueTensorStorage = typename CollectiveEpilogue::TensorStorage;
       MainloopTensorStorage mainloop;
       EpilogueTensorStorage epilogue;
     } tensors;
};
```
Otherwise, there is no need for Union and it should be simply like this
```c++
struct SharedStorage {   
      using EpilogueTensorStorage = typename CollectiveEpilogue::TensorStorage;
      EpilogueTensorStorage tensors;
 };
```

Reply:
Fixed this.